templates/ocs-deployment/temp.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
This file should not be part of the PR.
If I understand correctly it is generated by the test and should be deleted as part of the teardown

Reply:
Correct :)